### PR TITLE
Add Apache v2 license

### PR DESCRIPTION
**Pull request by leplatrem**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -25,6 +25,7 @@
       version='0.3.dev0',
       description='cliquet',
       long_description=README,
+      license='APLv2.0',
       classifiers=[
           "Programming Language :: Python",
           "Topic :: Internet :: WWW/HTTP",
```
Comment (Natim) [inline setup.py]: ``License :: OSI Approved :: Apache Software License``

File: CONTRIBUTORS.rst [hunk 2]
```diff
@@ -0,0 +1,8 @@
+Contributors
+############
+
+* Alexis Metaireau <alexis@mozilla.com>
+* Mathieu Leplatre <mathieu@mozilla.com>
+* Nicolas Perriault <nperriault@mozilla.com>
+* Rémy Hubscher <natim@mozilla.com>
```
Comment (Natim) [inline CONTRIBUTORS.rst]: ``rhubscher@mozilla.com``

File: setup.py (hunk 1 above)
Comment (ametaireau) [inline setup.py]: This is version 1 I believe, and I don't see the version 2 [in the list of trove classifiers](https://pypi.python.org/pypi?%3Aaction=list_classifiers). Maybe we should ask them to add one and in the meantime just add a "v2" at the end.

**Comment by ametaireau** [comment]
LGTM.

File: setup.py (hunk 1 above)
Comment (leplatrem) [inline setup.py]: Just looked at https://pypi.python.org/pypi/apache-libcloud :) simply mentions apache license (2.0)

**Pull request closed without merging**